### DISCUSSION TITLE
Avoid Querying/Searching if the Tracker is Down

### DISCUSSION THREAD

**Discussion by mouzzampk2014-zz** [cross-seed/cross-seed#102] [Q&A] [answered]
Hi @mmgoodnow, everything is working great. I have a suggestion if that can be implemented, is it possible to avoid searching the tracker for time being if the tracker is down. 

Thank you

**Comment by mmgoodnow** [comment, marked as answer]
This functionality is built in to Jackett.

**Reply by mouzzampk2014-zz** [reply]
Thank you but I am still seeing in `jackett` logs that `jackett` is searching the tracker every time `cross-seed` send the request unless I am missing the feature in `jackett`. Thank you

**Reply by mmgoodnow** [reply]
Hmm actually I think I was wrong about that. Might be a good idea especially because jackett queries tend to take up to 30 seconds when one of the trackers is down.

**Reply by mouzzampk2014-zz** [reply]
What I noticed with couple of trackers are taking around 1m or 1m30 but yes as you said taking longer. Would be great if its in `cross-seed`. Thanks

**Reply by mmgoodnow** [reply]
Here's the jackett issue: https://github.com/Jackett/Jackett/issues/10466

Please open a cross-seed issue for this

**Reply by mouzzampk2014-zz** [reply]
Done and thank you
